specs/Deployment/DeploymentAPI.md

```diff
@@ -179,12 +186,43 @@ they shut down, to refer to the updated framework package.
     }
 ```
 
+## Repair
+
+This API performs a status check, and if the main and singleton packages are already installed at
+the required version, it will attempt to repair those packages. All information about the version,
```

Comment:
So if the installer's got packages v1.2.3.4 and v1.2.3.4+ are already installed this will do a repair, but if v1.2.3.3 (or older) is installed this will...do its install work as usual?

Reply:
No. It is intended to not install anything new. I take the word "repair" means repair something that is already there. The API works only on already installed packages of the same WinAppSDK version that the framework package (that contains the current DM) belongs to.

In the scenario you asked, it will fail saying no packages installed for this version to be repaired.  Thanks for the cue, it makes sense to explicitly state that here.

Btw, do you disagree with my take on Repair ?

Reply:
As per offline continuation of this conversation, Initialize and Repair APIs would work on highest servicing revision of framework package installed for given WindowsAppSDK version.